docs/cloud-native-security/cspm-benchmark-rules.asciidoc

```diff
@@ -1,8 +1,13 @@
 [[cspm-benchmark-rules]]
-= Benchmark rules
-The Benchmark Integrations page lets you view the cloud security posture (CSP) benchmark rules for the <<cspm, Cloud security posture management>> (CSPM) and <<kspm, Kubernetes security posture management>> (KSPM) integrations.
+= Benchmarks
```

Comment:
We should probably also update **Benchmark Integrations** to **Benchmarks** on the [Elastic Security UI page](https://github.com/elastic/security-docs/blob/494ce028cece50b0d4696f3d0b0991f5c0742982/docs/getting-started/security-ui.asciidoc?plain=1#L95).